docs/reference/contributing/target/sai.md

```diff
@@ -35,7 +27,7 @@ Transfer and free functions, plus an init function, comprise this API.
 - `sai_init()` returns `SAI_RESULT_CONFIG_UNSUPPORTED` if the device can never support this configuration.
 - `sai_init()` returns `SAI_RESULT_CONFIG_MISMATCH` if the device is not able to support this configuration now because of other 'live' constraints, such as a shared format or clock configuration with a sibling.
 - `sai_free()` does nothing if passed a NULL pointer.
-- `sai_free()` deinitializes and unclocks the unused part of the device.
+- `sai_free()` deinitializes and disable associated clocks if the peripheral is not used anymore.
```

Comment:
I don't have permissions to fix this anymore. Please change this to "...and disables associated clocks if the peripheral is no longer in use."